src/main/scala/io/qbeast/context/QbeastContext.scala

```diff
@@ -126,8 +126,12 @@ object QbeastContext
     new QbeastContextImpl(config, keeper, indexedTableFactory)
   }
 
-  private def createKeeper(config: SparkConf): Keeper = Keeper(
-    config.getAll.filter(_._1.startsWith("spark.qbeast.keeper")).toMap)
+  private def createKeeper(config: SparkConf): Keeper = {
```

Comment:
The implementation is not specified in SparkConf. The Service Provider Interface is used in https://github.com/Qbeast-io/qbeast-spark/blob/5d9e5d6746edf1d512da1c34b0afa57bb79873fd/core/src/main/scala/io/qbeast/core/keeper/Keeper.scala#L124 

If the library for the `qbeast-spark-keeper` is added to the jars, it will load the implementation to use. The user should only configure the **host** and **port**.

Reply:
Cool :) Is it documented somewhere?